lib/std/crypto/blake2.zig

```diff
@@ -385,7 +385,7 @@ pub fn Blake2b(comptime out_bits: usize) type {
         pub const key_length_min = 0;
         pub const key_length_max = 64;
         pub const key_length = 32; // recommended key length
-        pub const Options = struct { key: ?[]const u8 = null, salt: ?[16]u8 = null, context: ?[16]u8 = null };
+        pub const Options = struct { key: ?[]const u8 = null, salt: ?[16]u8 = null, context: ?[16]u8 = null, expected_out_bits: usize = out_bits };
```

Comment:
Why store count in bits instead of bytes?
If you had: `expected_out_bytes: u32 = out_bits >> 3` then you wouldn't need the runtime right shift, or the `@intCast`.

Reply:
The comptime value the type itself is given in bits. Expressing the option in bytes and the type in bits would be an instant source of bugs for applications.